### ISSUE TITLE
Show Deleted Text

### ISSUE THREAD

**Issue by phrak** [sneakypete81/updatescanner#929]
Hi @sneakypete81 ,
Thanks for all your hard word - I love the Update Scanner plugin!

Is there a way to highlight deleted text in the comparison please?
This would be really useful to see when content has been removed from a page with the deleted content highlighted in red.

Ideally this would be an optional view so that users can ignore the deleted text if they don't want it.

Thanks in advance for your consideration and keep up the great work!

**Comment by yringot** [comment]
Yes, that would be great! it's possible to do it manually by taking the raw HTML of the old and new page (in the Debug Info menu item) and diff them, but it would be great if there was an easier way. Maybe just strikethrough and red paragraph background?
